src/Symfony/Component/Security/Csrf/EventListener/IsCsrfTokenValidAttributeListener.php

```diff
@@ -9,17 +9,17 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\Security\Http\EventListener;
+namespace Symfony\Component\Security\Csrf\EventListener;
```

Comment:
Unfortunately, it isn't this simple, we need to keep it in both places, deprecating the one in http. See #46094 for how we did this with the `Security` helper.